TreeDB/collections/column_vector_graph_manifest.go

```
@@ -0,0 +1,605 @@
+	rootID := catalog.rootID(collectionColumnManifestRootName(catalog.meta.Name))
+	if rootID == 0 {
+		status.State = VectorIndexStateColumnGraphRebuildNeeded
+		status.Reason = VectorIndexReasonColumnGraphRebuildNeeded
+		status.RebuildNeeded = true
+		return status, nil
+	}
+	if err := validateColumnManifestIdentityAtRoot(snap, rootID, *cfg.ActiveManifest); err != nil {
+		status.State = VectorIndexStateColumnGraphUnavailable
+		status.Reason = VectorIndexReasonColumnGraphCorrupt
+		status.RebuildNeeded = true
+		return status, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 408) but it returns nil

(nilerr)